api/app/dao/provider_details_dao.py

```diff
@@ -27,20 +26,10 @@ def get_provider_details_by_identifier(identifier):
 
 def get_alternative_sms_provider(identifier):
     alternate_provider = None
-    if os.getenv('FEATURE_SAP_ENABLED'):
```

Comment:
one last thing -- can we remove references to `FEATURE_SAP_ENABLED` from `api/manifest.yml` and `test_provider_details_dao`

Reply:
done